nuxeo-features/nuxeo-automation/nuxeo-automation-core/src/main/java/org/nuxeo/ecm/automation/core/operations/execution/RunOperationOnListInNewTransaction.java

```diff
@@ -94,10 +94,13 @@ public void run() throws OperationException {
             TransactionHelper.startTransaction();
```

Comment:
This could also be updated to use `TransactionHelper.runInNewTransaction` I suppose, like you did for `RunInNewTransaction.java`